File: Programming Languages/pub/sum/csharp/Sum.cs
// C# sum program

using System;

class Sum {

    public static int sum(int[] seq) {
	int s=0;
	for (int i=0; i<seq.Length; i++)
	    s+=seq[i];
	return s;
    }

    public static void Main() {
	int[] seq=new int[] {5,6,1,8,3,7};
	Console.WriteLine(sum(seq));
    }

}

File: Programming Languages/pub/sum/go/sum.go
/* Go sum program */

package main

import ("fmt")

func sum(seq []int) int {
    s:=0
    for i:=0; i<len(seq); i++ {
        s+=seq[i]
    }
    return s;
}

func main() {
    seq:=[]int{5,6,1,8,3,7}
    fmt.Printf("%d\n",sum(seq))
}

